gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/Project.java

```diff
@@ -157,10 +157,10 @@ public Project reload() {
    * completes, the project is not retrievable by the {@link ResourceManager#get} and
```

Comment:
Unrelated to this PR. The Builder projectId, removeLabel, clearLabels (and parent, which is not that important) seems not to be covered by unit-test